pep-0630.rst

```diff
@@ -466,6 +525,14 @@ its variable-size storage is currently consumed by slots. Fixing this
 is complicated by the fact that several classes in an inheritance
 hierarchy may need to reserve some state.
 
+Lossless conversion to heap types
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The heap type API was not designed for “lossless” conversion from static types.
```

Comment:
Could you reword this with a clearer language?

Reply:
```suggestion
Lossless conversion to heap types
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The heap type API was not designed for “lossless” conversion from static types,
that is creating a type that works exactly like a given static type.
```